code/modules/recycling/disposal.dm

```diff
@@ -133,6 +133,15 @@
 			update()
 			return
 
+		if(istype(I, /obj/item/weapon/storage/bag/chem))

```

Comment:
Same as my previous note, if this actually works as is, then you can just have it combined with the trash bag check above.